docs/dev/api.rst

```diff
@@ -25,60 +40,178 @@ The Library Class
 
     .. automethod:: get_album
 
-    .. automethod:: add
-
-    .. automethod:: add_album
+    Any modifications must go through a :class:`Transaction` which you get can
+    using this method:
 
     .. automethod:: transaction
 
-Transactions
-''''''''''''
-
-The :class:`Library` class provides the basic methods necessary to access and
-manipulate its contents. To perform more complicated operations atomically, or
-to interact directly with the underlying SQLite database, you must use a
-*transaction*. For example::
-
-    lib = Library()
-    with lib.transaction() as tx:
-        items = lib.items(query)
-        lib.add_album(list(items))
-
-.. currentmodule:: beets.dbcore.db
+.. _SQLite: http://sqlite.org/
+.. _ORM: http://en.wikipedia.org/wiki/Object-relational_mapping
 
-.. autoclass:: Transaction
-    :members:
 
 Model Classes
 -------------
 
 The two model entities in beets libraries, :class:`Item` and :class:`Album`,
-share a base class, :class:`Model`, that provides common functionality and
+share a base class, :class:`LibModel`, that provides common functionality and
 ORM-like abstraction.
```

Comment:
This is a detail that might not matter, but `dbcore.db.Model` provides that ORM-like abstraction. `LibModel` provides shared functionality between `Album` and `Item` that is more music-specific.

Reply:
I was trying to simplify the description but I agree this was imprecise.